src/iconsManifest/supportedExtensions.ts

```diff
@@ -4552,5 +4552,10 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
       disabled: true,
     },
+    {
+      icon: 'ndst',
+      extensions: ['ndst.yaml', 'ndst.yml', 'ndst.json'],
```

Comment:
Are these extensions or filenames? Just to be sure.

Reply:
Are extensions.